Sources/Orbit/Components/InputField.swift

```diff
@@ -71,42 +75,18 @@ public struct InputField: View {
     }
 
     @ViewBuilder var input: some View {
-        if isSecure {
-            secureField
-        } else {
-            textField
+        switch mode {
+            case .actionsHandler(let onEditingChanged, let onCommit, let isSecure):
+                if isSecure {
+                    secureField(onEditingChanged: onEditingChanged, onCommit: onCommit)
+                } else {
+                    textField(onEditingChanged: onEditingChanged, onCommit: onCommit)
+                }
+            case .formatter(let formatter):
+                TextField("", value: $value, formatter: formatter)
```

Comment:
According to docs, the formatter will be used when the field value is confirmed (probably the `onCommit` phase, by enter key, keyboard dismiss...). 

In that case, when it would not work in realtime, is there still any advantage from simply letting caller do whatever formatting it needs at the `onCommit` phase?